src/std/ide/swank/autodoc.ss

```diff
@@ -0,0 +1,93 @@
+(import :gerbil/expander ./message :std/sugar
+	./context ./api :std/format
+	:std/srfi/13)
+(export #t)
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Completions, Autodoc and Signature
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;; search for a procedure in 0gambit-procedures
+;; returns the procedure symbol if it finds it
+
+(gxc#optimizer-info-init!)
```

Comment:
please import `:gerbil/compiler/optimize-base`, you can't just use this without ensuring the module that provies its implementation is loaded.